src/tile.cpp

```diff
@@ -1438,7 +1426,7 @@ void Tile::internalAddThing(uint32_t, Thing* thing)
 		CreatureVector* creatures = makeCreatures();
 		creatures->insert(creatures->begin(), creature);
 	} else {
-		Item* item = thing->getItem();
+		auto item = thing->getItem();
```

Comment:
Could this be `const auto`
```suggestion
		const auto item = thing->getItem();
```